counterecryptor_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"math/rand"
 	"testing"
 
-	"github.com/elazarl/goproxy"
+	"github.com/mach-spotqa/goproxy"
```

Comment:
We should keep this as it is for this repo, right?

Reply:
My bad. Included it on PR by mistake. But it should fix compile error as new PR #378